README.md

```diff
@@ -1 +1,66 @@
-# capstone
\ No newline at end of file
+# capstone
+
+ This is a project for a climate change summit website. It contains details about the conference such as the speakers, the date, venue and the main speakers.
+
+
+## Built With
+
+- HTML/CSS/JS
+
+## Getting Started
+
+- Click on the clone button to copy the clone
+- Open Terminal and run git clone <copied address>
+- Clone the repo
+- From you terminal run " code ." to open in your code editor.
+
+## Sneak peek to my Capston Project
+
+- Portfolio: [Alaa Shaheen](https://github.com/AlaaShaheen2/capstone)
+
+
+## Authors
+👤 **Simon Gideon**
+👤 **Alaa Shaheen**
+
+- GitHub: [@AlaaShaheen2](https://github.com/AlaaShaheen2)
+- Twitter: [@Alaa Shaheen](https://twitter.com/AlaaShaheen93)
+- LinkedIn: [@Alaa Shaheen](https://www.linkedin.com/in/alaa-shaheen-879140240/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Video Description 
+The video description of the  project can be accessed using [this loom link](https://www.loom.com/share/a90404d1d1f040e28e4eb93dd5cfd3c5)
+
+### Setup
+- copy this link `https://github.com/AlaaShaheen2/capstone`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write ` https://github.com/AlaaShaheen2/capstone`.
+- go to the repository folder in your command prompt and open `index.html` file
+### Install
+- write `https://github.com/AlaaShaheen2/capstone`.
+- go to the repository folder in your command prompt and open `index.html` file
+### Usage
+- By clicking on 'Go live' in VS studio, you can be able to see this project in action.
+- You can also access this open source website in the aboue provided demo link
+### Run tests
+- You can navigate to different parts of the page from the navigation bar and you can also inspect using the available browser tools to se how the webpages were built.
+### Deployment
+- The project is deployed at [ https://alaashaheen2.github.io/capstone/]( https://alaashaheen2.github.io/capstone/)
+
+
+## Acknowledgments
+
+- Thanks to the reviewers
+- Thanks to Cindy Shin the author of the original desig
```

Comment:
- [ ] Like the last code reviewer said kindly add a design link. You just added text.

Reply:
@ahmetbozaci done